config.m4

```diff
@@ -5,7 +5,7 @@ PHP_ARG_ENABLE(quickhash, whether to enable quickhash support,
 [  --enable-quickhash      Enable quickhash support])
 
 if test "$PHP_QUICKHASH" != "no"; then
-  PHP_NEW_EXTENSION(quickhash, quickhash.c qh_inthash.c qh_intset.c qh_intstringhash.c qh_stringinthash.c qh_iterator.c lib/quickhash.c lib/hash-algorithms.c lib/iterator.c, $ext_shared)
+  PHP_NEW_EXTENSION(quickhash, quickhash.c qh_inthash.c qh_intset.c qh_intstringhash.c qh_stringinthash.c qh_iterator.c lib/quickhash.c lib/hash-algorithms.c lib/iterator.c, $ext_shared,,-DZEND_ENABLE_STATIC_TSRMLS_CACHE=1)
```

Comment:
What does this static cache thingy do? I've not had to use it for any other extension, including Xdebug.